docs/binary.md

```diff
@@ -411,7 +411,7 @@ If the byte is `00`, a `CFrame` looks like this:
 | Field Name  | Format                  | Value                                                                                                        |
 |:------------|:------------------------|:-------------------------------------------------------------------------------------------------------------|
 | ID          | `u8`                    | Always `00` in this case.                                                                                    |
-| Orientation | Array of 9 `f32` values | The rotation matrix of the `CFrame`. It represents the RightVector, UpVector, and LookVector, in that order. |
+| Orientation | Array of 9 `f32` values | The rotation matrix of the `CFrame` in **row-major order**: the XVector, YVector, and ZVector, in that order.|
```

Comment:
I don't feel that specifying "in row-major order" is actually helpful to would-be implementers. If we want to be specific, we can indicate the order the components are in (using the all too popular `r00` through `r22`) but I think just saying that it's the `XVector`, `YVector`, and `ZVector` should be enough for anyone familiar with Roblox's ecosystem.